.env

```diff
@@ -0,0 +1,4 @@
+VITE_PIPED_API=https://pipedapi.kavin.rocks
+VITE_PIPED_PROXY=https://pipedproxy.kavin.rocks
+VITE_PIPED_INSTANCES=https://piped-instances.kavin.rocks/
+VITE_BASE_URL=/
```

Comment:
I think `VITE_BASE_PATH` is more appropriate for this env variable.

Reply:
So I realised that setting the vite base path from this env file doesn't actually work. Apparently variables loaded from env files aren't available yet in the config file. However, I found out you can do `vite build --base=/whatever` to override the base setting which is even better than an environment variable.

The uses in the code itself have to stay `BASE_URL` because that's the variable vite exports to access its base path: https://vitejs.dev/guide/build#public-base-path